packages/tldraw/src/lib/ui/hooks/useExportAs.ts

```diff
@@ -97,3 +97,13 @@ export function useExportAs() {
 		[editor, addToast, msg]
 	)
 }
+
+function getTimestamp() {
+	const now = new Date()
+
+	const hours = String(now.getHours()).padStart(2, '0')
+	const minutes = String(now.getMinutes()).padStart(2, '0')
+	const seconds = String(now.getSeconds()).padStart(2, '0')
+
+	return `${hours}-${minutes}-${seconds}`
+}
```

Comment:
Let's base this on how macos formats screenshot titles which is `Screenshot at YY-MM-DD HH.MM.SS`

This way they'll sort correctly if you do exports on different days